include/wlr/types/wlr_output_damage.h

```diff
@@ -59,8 +60,8 @@ void wlr_output_damage_destroy(struct wlr_output_damage *output_damage);
  * `wlr_output_damage_swap_buffers` needs to be called. The region of the output
  * that needs to be repainted is added to `damage`.
  */
-bool wlr_output_damage_make_current(struct wlr_output_damage *output_damage,
-	bool *needs_swap, pixman_region32_t *damage);
+bool wlr_output_damage_begin(struct wlr_output_damage *output_damage,
+	struct wlr_renderer *renderer, bool *needs_swap, pixman_region32_t *damage);
```

Comment:
The renderer can be obtained from the output.

Reply:
oh we don't even need the renderer anymore, good catch.